fuel-relayer/Cargo.toml

```diff
@@ -35,7 +35,7 @@ serde = "1.0"
 serde_json = "1.0"
 sha3 = "0.10"
 thiserror = "1.0"
-tokio = { version = "1.20", features = ["macros"]}
+tokio = { version = "1.21", features = ["macros"]}
```

Comment:
TIL relayer was on an entirely different version of tokio then the rest of the client, wonder if this speeds up compile times